arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -1247,115 +1165,81 @@
   "definitions": {
     "CheckAvailabilityParameters": {
       "properties": {
-        "id": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource Id"
-        },
-        "name": {
-          "type": "string",
-          "description": "Resource name"
-        },
-        "type": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource type"
-        },
-        "location": {
-          "type": "string",
-          "description": "Resource location"
-        },
-        "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "Resource tags"
-        },
-        "sku": {
-          "$ref": "#/definitions/Sku",
-          "description": "The sku of the created namespace"
-        },
         "isAvailiable": {
           "type": "boolean",
-          "description": "True if the name is available and can be used to create new Namespace/NotificationHub. Otherwise false."
+          "description": "True if the name is available and can be used to create new namespace/notification hub, otherwise false"
         }
       },
       "required": [
-        "name",
-        "location"
+        "name"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
       ],
-      "description": "Parameters supplied to the Check Name Availability for Namespace and NotificationHubs."
+      "description": "Parameters supplied to the check name availability for namespace and notification hubs"
     },
     "CheckAvailabilityResult": {
       "properties": {
         "isAvailiable": {
           "type": "boolean",
-          "description": "True if the name is available and can be used to create new Namespace/NotificationHub. Otherwise false."
+          "description": "True if the name is available and can be used to create new namespace/notification hub, otherwise false"
```

Comment:
I think it's given at this state that user has previously taken this name. We can queue up a bug request but this is a user experience improvement rather than doc issue. If I update it here, it will just not align with our code base.